benchmarks/base.py

```diff
@@ -18,9 +19,14 @@ def sources_dir():
     return path
 
 
+@lru_cache(None)
+def get_data_for_file(path, file_size):
```

Comment:
```suggestion
@lru_cache(None)
def get_data_for_file(path, file_size):
    # This function will generate the same data for the same path,
    # which would allow to make datasets grow on top of the existing
    # ones. The `path` is a key to this function's caching system, and
    # even if it is not used, it needs to be present in the signature to
    # allow cache lookups.
```

Reply:
@pared how about this?